ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-ranGen.yaml

```diff
@@ -184,3 +184,19 @@ spec:
 #                user:
 #                  name: root
 #     ---- sources needed for updating CRI-O workload-partitioning ends here ----
+
+    # ---- Sources needed to configure IPsec encryption for external traffic ----
+    # 1. Set the `$mode` in the `enable-ipsec.yaml`
+    # 2. Provide the following certificate files:
+    #    * left_server.p12
+    #    * ca.pem
+    # 3. Set the following values in the `ipsec-endpoint-config.yml`:
+    #    * `$clusterNode` - the host name or IP of the cluster node for the IPsec tunnel on the cluster side
```

Comment:
since this is for SNO
can be just `$node` 
note that `left` should be reachable/resolvable
so it's the IP address or DNS hostname (not host name)

Reply:
I was thinking the source-crs can be generic for both MNO and SNO, so maybe keep it `$clusterNode`